test/server/util-test.js

```diff
@@ -355,4 +359,39 @@ describe('Utils', () => {
             });
         });
     });
+
+    it('Build Chrome arguments', () => {
+        const config = {
+            userProfile: false,
+            headless:    false,
+            userArgs:    ''
+        };
+
+        const cdpPort      = '';
+        const platformArgs = '--no-first-run';
+
+        const tempProfileDir = {
+            path: '/temp/testcafe/chrome-profile-34904xxzNmO5Vkbtz'
+        };
+
+        let chromeArgs = '';
+
+        const IN_DOCKER_FLAGS_RE = new RegExp(IN_DOCKER_FLAGS.join(' '));
+        const NO_SANDBOX_FLAG_RE = new RegExp('--no-sandbox');
```

Comment:
Maybe it's better to just add a RegExp for `--disable-dev-shm-usage`. It will make tests more illustrative and enable checking for this flag in the last test.